Post tutorial fixup PR (#25484)
idaholab/moose: tutorials/darcy_thermo_mech/doc/content/workshop/numerical/fem_overview.md

!---

## FEM can be used to solve both linear and nonlinear PDEs

FEM is a method for numerically approximating the solution to [!ac](PDEs).
FEM is widely applicable for a large range of PDEs and domains.

Example PDEs:

> Have you seen them before? Are they linear/ nonlinear? Coupled?

Reviewer: ```suggestion
Have you seen them before? Are they linear/nonlinear? Coupled?
```

1.  Integrate by parts and use the divergence theorem to get the desired derivative order on your
    functions and simultaneously generate boundary integrals.

!alert note title=Exercise

> Obtain the weak form for the equations listed before, write the equations in terms of the coefficients

Author: ```suggestion
Obtain the weak form for the equations listed on the previous slide
```
